src/Form/Type/CustomBlockType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Lakion\SyliusCmsBundle\Form\Type;
+
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class CustomBlockType extends AbstractResourceType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options = [])
+    {
+        $builder
+            ->add('name', TextType::class, [
```

Comment:
Any of them can fit in a single line, but there's no need for that; right now it's just the matter of personal preferences.